examples/log/boxfddp_vs_boxddp.log

```diff
@@ -23,68 +23,68 @@ iter 	 cost 	      stop 	    grad 	  xreg 	      ureg 	 step 	 feas
   19  3.09833e+05  1.20330e+08  1.91734e+06  1.00000e-01  1.00000e-01   0.2500     0
 iter 	 cost 	      stop 	    grad 	  xreg 	      ureg 	 step 	 feas
   20  2.34230e+05  1.70003e+08  1.14113e+06  1.00000e-01  1.00000e-01   0.5000     0
-  21  1.71141e+05  2.00479e+08  5.90291e+05  1.00000e-01  1.00000e-01   0.2500     0
```

Comment:
There is something in the way that we are using the box solvers that is causing this difference in numerical behaviour. The difference in cost here is quite high (order of 1.). This is in the same vein as the difference because of different architectures, as reported in https://github.com/loco-3d/crocoddyl/pull/696.
I'm inclined to say it is because of `box-qp::Hff_inv_llt_` , which is an Eigen::LLT object. But this is only my guess.

Reply:
You might be right. We were discussing about it.
@wxmerkt you might want to be involved in this comment.

Reply:
I have seen some issues with LLT, and particularly the pre-allocated Hff_inv_llt_ (`Hff.llt().solve()` is fine) in _some_ cases on a wider benchmark. Changing to a different decomposition solved it indeed.

Reply:
I think this is an interesting topic, but I would prefer to handle it in a dedicate PR.

We need to evaluate carefully the proposition of using LDLT. Note that its limitation is the computational speed (LLT is faster). I have conducted a benchmark, you could  check more information here: https://github.com/loco-3d/crocoddyl/issues/743#issuecomment-621667401.

Let's continue this discussion in that thread :)